packages/extension/README.md

```diff
@@ -33,10 +33,16 @@ npm run test
 
 ## Debugging
 
+### In Chrome
+
 1. Open Chrome
 2. Go into Extensions
 3. Check **Developer Mode**
 4. Click **Load unpacked extension...**
 5. Choose **packages/extension/dist** directory
 6. Click **background page** to debug `background.js`
 7. Click **Reload (⌘R)** to pull in changes to `manifest.json`
```

Comment:
we need to update this since `3.5.0` this is no longer the case since we use the CDP